### PR TITLE
implementation of support for loading stories from multiple files

### PR DESCRIPTION
**Pull request by tmbo**
**Proposed changes**:
- instead of specifying a single file for stories, this allows to load stories from a directory of story files
- in the future we can think about specifying multiple files directly (instead of a folder) - but this was easier to do for now

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [x] added some tests for the functionality
- [x] updated the documentation
- [x] updated the changelog

### CODE REVIEW

File: rasa_core/training/dsl.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 import json
 import logging
 import os
+import rasa_nlu
```
Comment (amn41) [inline rasa_core/training/dsl.py]: we need to specifically import this submodule, let's do:
`from rasa_nlu import utils as nlu_utils`

**Comment by Ghostvv** [comment]
trainer also reads hidden file on my mac .DS_Store

**Comment by hotzenklotz** [comment]
Very nice :-) I am looking forward to this.

**Comment by tmbo** [comment]
I've address the hidden file issue & your comment @amn41 

mind taking another look?

**Comment by tmbo** [comment]
https://github.com/RasaHQ/rasa_nlu/pull/904 needs to be merged into nlu before this can be merged.

**Pull request merged**